migration/migrating_3_4/premigration-checklists.adoc

```diff
@@ -0,0 +1,97 @@
+* [ ] External applications, services, and appliances that use services provided by the cluster have the correct network configuration and permissions to access the cluster.
+* [ ] Internal container image dependencies are met.
++
+If an application uses an internal image in the `openshift` namespace that is not supported by {product-title} {product-version}, you can manually update the xref:../../migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc#migration-updating-deprecated-internal-images_migrating-3-4[{product-title} 3 image stream tag] with `podman`.
+* [ ] All xref:../../migration/migrating_3_4/migrating-applications-with-cam-3-4.adoc#migration-prerequisites_migrating-3-4[{mtc-short} prerequisites] are met.
+* [ ] The xref:../../authentication/understanding-identity-provider.adoc#supported-identity-providers[identity provider] is working.
+
+[id="resource-capacity-checklist_{context}"]
+== Resource capacity checklist
+
+* [ ] All clusters have sufficient resources for migration.
++
+[NOTE]
+====
+Clusters require additional memory, CPUs, and storage in order to run a migration on top of normal workloads. Actual resource requirements depend on the number of Kubernetes resources being migrated in a single migration plan. Migrations must be run in a test environment in order to estimate the resource requirements.
+====
+
+* [ ] The source cluster meets the link:https://docs.openshift.com/container-platform/3.11/install/prerequisites.html#hardware[minimum hardware requirements] for {product-title} 3.
+* [ ] The target cluster meets the minimum hardware requirements for the specific platform and installation method, for example, on xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#minimum-resource-requirements_installing-bare-metal[bare metal], for {product-title} {product-version}.
+* [ ] The target cluster has xref:../../storage/dynamic-provisioning.adoc#defining-storage-classes_dynamic-provisioning[storage classes] defined for the storage types, for example, block volume, file system, or object storage, used by the source cluster.
+* [ ] The xref:../../storage/understanding-persistent-storage.adoc#pvc-storage-class_understanding-persistent-storage[default storage class] of the source and target clusters is the same.
+* [ ] If the source cluster uses the link:https://docs.openshift.com/container-platform/3.11/architecture/networking/sdn.html#network-isolation-multitenant[ovs-multitenant plug-in] to achieve network isolation, the network visibility between namespaces is correctly configured on the target cluster.
++
+By default, {product-title} {product-version} uses an xref:../../networking/network_policy/about-network-policy.html[open network policy]. All pods in a project are accessible from other pods and network endpoints.
+* [ ] If you are migrating more than 20 TB, the target cluster and the replication repository have sufficient storage space.
```

Comment:
@apinnick I am confused why it's 20 TB. No matter how much data, customer have to ensure the target cluster and the replication repository have sufficient storage space.

Reply:
True. I can remove the value.